Makefile

```diff
@@ -313,7 +313,9 @@ create-cluster: $(CLUSTERCTL) ## Create a development Kubernetes cluster on AWS
 		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
 		wait --for=condition=Ready --namespace=cert-manager --timeout=15m pods --all
 	# Wait for webhook servers to be ready to take requests
-	sleep 10
+	kubectl \
+		--kubeconfig=$$(kind get kubeconfig-path --name="clusterapi") \
+		wait --for=condition=Available --timeout=5m apiservice v1beta1.webhook.cert-manager.io
```

Comment:
Can we remove the sleep on line 310 and the wait on lines 312-314 as well with the addition of this wait?

Reply:
Probably. I don't have any of these other sleeps/waits in my Tiltfile and I've never had issues deploying.